Manager/src/main/java/io/raspberrywallet/manager/common/readers/WalletUUIDReader.java

```diff
@@ -0,0 +1,46 @@
+package io.raspberrywallet.manager.common.readers;
+
+import lombok.Synchronized;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.UUID;
+
+public class WalletUUIDReader extends Reader<UUID> {
```

Comment:
What is the Wallet UUID ?

Reply:
I had to choose idntifier for every wallet on server side. I've decided to use UUID, that is random for every wallet. The secret on the server side is stored as key-value pair, where key is wallet UUID.

Reply:
Ok, looks good